ts/features/idpay/onboarding/screens/InitiativeDetailsScreen.tsx

```diff
@@ -119,17 +51,17 @@ const InitiativeDetailsScreen = () => {
     });
   }, [machine, serviceId]);
 
+  const initiative = useSelector(machine, selectInitiative);
   const isAcceptingTos = useSelector(machine, isUpsertingSelector);
   const isLoading = useSelector(machine, isLoadingSelector);
-  const description = useSelector(machine, initiativeDescriptionSelector);
-
-  const hasDescription = description !== undefined;
 
   const [needsScrolling, setNeedsScrolling] = React.useState(true);
   const [hasScrolled, setHasScrolled] = React.useState(false);
+
   const scrollViewHeightRef = React.useRef(0);
   const isContinueButtonDisabled =
-    isLoading || (hasDescription && needsScrolling && !hasScrolled);
+    isLoading || (needsScrolling && !hasScrolled);
```

Comment:
Have you tested what happens when the initiative has no description?

Reply:
It is unlikely that initiatives will come without a description. But this is a good point to discuss in https://github.com/pagopa/io-app/pull/4541